src/commands/test/lib/nunit-xml-util.ts

```diff
@@ -106,28 +129,51 @@ export class NUnitXmlUtil extends XmlUtil {
   }
 
   combineTestResultsAttribute(xml1: Document, xml2: Document, attributeName: string) {
```

Comment:
Should this method be renamed to `combineResultsAttribute` to fit in with the other renamed methods?